tests/python/relay/test_error_reporting.py

```diff
@@ -0,0 +1,27 @@
+import tvm
+from tvm import relay
+
+def check_type_err(expr, msg):
+    try:
```

Comment:
We will need to have an example on type inference error, because that is the most crucial use case of error reporting

Reply:
+1

Reply:
What do you mean? the test cases are only focused on type inference issues.